azure-pipelines.yml

```diff
@@ -17,7 +17,8 @@ stages:
         timeoutInMinutes: 90
         displayName: 'Validation'
         variables:
-          buildOutputLocation: 'D:\tools'
+          - name: DiffFile
+            value: '$(Build.ArtifactStagingDirectory)\format.diff'

```

Comment:
Can/should this use a forward slash? It appears that we prefer to use forward slashes except when passing things to command prompt scripts that want backslashes.